app/src/main/java/co/nimblehq/ui/second/SecondActivity.kt

```diff
@@ -61,11 +55,19 @@ class SecondActivity : BaseActivity() {
         Toast.makeText(this, "Permission camera never ask", Toast.LENGTH_SHORT).show()
     }
 
-    override fun onRequestPermissionsResult(requestCode: Int, permissions: Array<out String>, grantResults: IntArray) {
+    override fun onRequestPermissionsResult(
+        requestCode: Int,
+        permissions: Array<out String>,
+        grantResults: IntArray
+    ) {
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
         onRequestPermissionsResult(requestCode, grantResults)
     }
 
+    private fun bindPersistedData(data: Data) {
+        persistTextView.text = data.content
```

Comment:
Should this be `tvPersist` ? Maybe out of scope

Reply:
@Tuubz I'll put TODO comment here to refactor naming later 🙆‍♂️